### ISSUE TITLE
Tiles are rotated incorrectly for orthographic rendererer

### ISSUE THREAD

**Issue by SolarLune** [lafriks/go-tiled#22]
Hello!

I believe tiles are being rotated incorrectly for the orthographic renderer - can you confirm if what I'm seeing is what you see with rotated tiles in a TMX map as well? If so, I can submit a PR. Here's an example of what I'm seeing with a relatively simple TMX file I have.

[rotationtest.zip](https://github.com/lafriks/go-tiled/files/4642656/rotationtest.zip)

Here's a screenshot showing the issue after rendering the result to an image and then drawing it to the screen using ebiten:

![Screenshot from 2020-05-18 00-26-19](https://user-images.githubusercontent.com/4733521/82185629-5e259f00-989e-11ea-9604-1c7fc202d44d.png)

The tiles are rotated improperly, though it seems that horizontally or vertically flipped tiles are transformed properly. The tiles are correct after I tweaked the OrthogonalRendererEngine.RotateTileImage function:

![Screenshot from 2020-05-18 00-26-04](https://user-images.githubusercontent.com/4733521/82185628-5d8d0880-989e-11ea-8a2a-a8d508954513.png)

Here's the change I made:

```go
// RotateTileImage rotates provided tile layer.
func (e *OrthogonalRendererEngine) RotateTileImage(tile *tiled.LayerTile, img image.Image) image.Image {
	timg := img
	if tile.HorizontalFlip {
		timg = imaging.FlipH(timg)
	}
	if tile.VerticalFlip {
		timg = imaging.FlipV(timg)
	}
	if tile.DiagonalFlip {
		//This was "timg = imaging.FlipH(imaging.Rotate270(timg))"
                 timg = imaging.FlipH(imaging.Rotate90(timg)) 
	}

	return timg
}

```

It seems like this is a rather simple solution to get the tiles to transform properly.

**Comment by lafriks** [comment]
Please submit PR

**Comment by SolarLune** [comment]
Hello! Sorry for the late reply; I submitted a PR for this - #23.

**Comment by lafriks** [comment]
Fixed in #22

**Comment by SolarLune** [comment]
Thanks!
